src/lib/pipeline.ml

```diff
@@ -158,7 +158,7 @@ module Construct = struct
 
   let star fastq = Star fastq
 
-  let hisat fastq = Hisat fastq
+  let hisat ?(configuration = Hisat.Configuration.default_v1) fastq = Hisat(configuration, fastq)
```

Comment:
If you'd like? My assumption was the default should be what was there already so nothing changes for any pipelines using v1